core/src/main/java/nl/nn/adapterframework/jdbc/migration/LiquibaseImpl.java

```diff
@@ -62,8 +60,7 @@ public LiquibaseImpl(IbisContext ibisContext, JdbcConnection connection, Configu
 	}
 
 	private void log(String message) {
-		if(ibisContext != null) 
-			ibisContext.log(configuration.getName(), null, message);
+		configuration.publishEvent(new ConfigurationMessageEvent(configuration, message));
```

Comment:
Kan je hiervoor een configuration.log(message) maken?
Die kan dan publishEvent(new ConfigurationMessageEvent(this, message) doen